src/packet/signature/config.rs

```diff
@@ -299,11 +299,14 @@ impl SignatureConfig {
         }
     }
 
-    /// Returns an iterator over all subpackets of this signature.
+    /// Returns an iterator over the hashed subpackets of this signature.
     pub fn subpackets(&self) -> impl Iterator<Item = &Subpacket> {
```

Comment:
I think if we do this, this method should be renamed to `hashed_subpackets`

Reply:
I'd say rename, and add a `subpackets` function marked with `#[deprecated(note = "Use hashed_subpackets")]` that delegates to hashed_subpackets.

Reply:
> I'd say rename, and add a `subpackets` function marked with `#[deprecated(note = "Use hashed_subpackets")]` that delegates to hashed_subpackets.

Good point. I did this in 31aa70a0ed197fb196886175be22e0b2b40b6f34 (and am happy to clean up the commit history/comments, if/when we arrive at a consensus for an appraoch)